### ISSUE TITLE
[Bug]: Increase Accessibility -lighthouse metric

### ISSUE THREAD

**Issue by dev-ansuman** [testorg-mindfire/InterDash#160]
## Describe the Bug

The web-app has a very low score of accessibility in lighthouse.
Increase the accessibility metric in the lighthouse - low accessbility score shows the inability of the web-app to be less accessible for the users

## Expected Behavior

The Accessibility score in the light house metic should be more than 95.

## Actual Behavior

The Accessibility score in the light house metic is currently ranging between 80s.
